packages/sources/web-components/src/components.d.ts

```diff
@@ -53,15 +82,39 @@ declare namespace LocalJSX {
     | 'ghost'
     | 'conversion';
     }
+    interface VtmnToggle {
+        /**
+          * The id of the toggle.
+          * @default 'my-id'
+         */
+        "id"?: string;
```

Comment:
For me it's required (as you saif in story), because if not, we will have several toggles with the same id in a HTML page... So you can remove the `?` and delete the `@default` value.